nemo/collections/nlp/models/machine_translation/megatron_nmt_model.py

```diff
@@ -399,21 +399,12 @@ def _log_multilingual_bleu_and_loss(self, dataloader_idx, bleu_score, loss, mode
         Function to log multilingual BLEU scores with the right source-target language string instead of just the dataloader idx.
         """
         # Check if one-many or many-one and log with lang ids instead of dataloader_idx
-        reverse_lang_direction = self._cfg.train_ds.reverse_lang_direction
         if isinstance(self.src_language, ListConfig):
-            translation_lang_string = (
-                f'{self.src_language[dataloader_idx]}-{self.tgt_language}'
-                if not reverse_lang_direction
-                else f'{self.tgt_language}-{self.src_language[dataloader_idx]}'
-            )
+            translation_lang_string = f'{self.src_language[dataloader_idx]}-{self.tgt_language}'
             self.log(f'{mode}_sacreBLEU_{translation_lang_string}', bleu_score, sync_dist=True)
             self.log(f'{mode}_loss_{translation_lang_string}', loss, sync_dist=True)
```

Comment:
These two lines are redundant and repeat in both if and else statements. I think it would be cleaner to move these out of statements.